src/types/onyx/ReportAction.ts

```diff
@@ -76,9 +78,9 @@ type ReportActionBase = {
     isLoading?: boolean;
 
     /** accountIDs of the people to which the whisper was sent to (if any). Returns empty array if it is not a whisper */
-    whisperedToAccountIDs?: number[];
+    whisperedToAccountIDs?: Array<number | undefined>;
```

Comment:
@kubabutkiewicz are you sure it's ok to have undefined in this array?

Reply:
Yeah, since `accountID` can be undefined, we should expect that in this array we can have undefined values

Reply:
Maybe we can filter it out? 
`whisperedToAccountIDs: isReceiptBeingScanned && currentUserAccountID ? [currentUserAccountID] : [],`